packages/trpc/server/routers/viewer.tsx

```diff
@@ -493,7 +493,19 @@ const loggedInViewerRouter = createProtectedRouter()
         },
         unconfirmed: {
           endTime: { gte: new Date() },
-          status: { equals: BookingStatus.PENDING },
+          OR: [
+            {
+              AND: [
+                { NOT: { recurringEventId: { equals: null } } },
+                {
+                  status: { equals: BookingStatus.PENDING },
+                },
+              ],
+            },
```

Comment:
```suggestion
            {
              recurringEventId: { not: null },
              status: { equals: BookingStatus.PENDING },
            },
```

We can simplify this